aspnetcore/grpc/diagnostics.md

```diff
@@ -57,19 +65,19 @@ info: Microsoft.AspNetCore.Hosting.Diagnostics[2]
       Request finished in 1.4113ms 200 application/grpc
 ```
 
-## Access server-side logs
+### Access server-side logs
 
 How you access server-side logs depends on the environment in which you're running.
```

Comment:
You can give this link as an example: https://docs.microsoft.com/azure/azure-monitor/app/diagnostic-search